src/encoder.rs

```diff
@@ -1868,6 +1867,33 @@ fn encode_partition_topdown(fi: &FrameInvariants, fs: &mut FrameState,
     }
 }
 
+use rayon::prelude::*;
+
+
+#[inline(always)]
+fn build_coarse_pmvs(fi: &FrameInvariants, fs: &FrameState) -> Vec<[Option<MotionVector>; REF_FRAMES]> {
+// initial coarse ME loop
+  assert!(!fi.sequence.use_128x128_superblock);
+  let sby_range = 0..fi.sb_height;
+  let sbx_range = 0..fi.sb_width;
+
+  let sbos = (sby_range).flat_map(|sby| {
+      sbx_range.clone().map(move |sbx| SuperBlockOffset { x: sbx, y: sby })
+  }).collect::<Vec<SuperBlockOffset>>();
```

Comment:
Naming the indices `x` and `y` seems to reduce the noise here.
```rust
.flat_map(|y| {
  sbx_range.clone().map(move |x| SuperBlockOffset { x, y })
})
```

Reply:
It is a good idea indeed :)